Cargo.toml

```diff
@@ -45,7 +47,7 @@ num-complex = "0.4"
 num-traits = "0.2.15"
 indenter = "0.2"
 regex-lite = "0.1.0"
-serde = "1.0"
+serde = { version = "1.0", features = ["derive"] }
```

Comment:
We only list the dependency in the top level Cargo.toml, add the features in the crate where it's needed (e.g. see the Cargo.toml in the wasm crate where it does what this line does).